### ISSUE TITLE
Fregot doesn't recognize raw string declaration

### ISSUE THREAD

**Issue by sean-zou** [fugue/fregot#139]
Wanna try out the nice debug feature of fregot, but getting the error. Looks like raw string isn't supported yet?
```fregot test policy
fregot (fatal parse error):
  "policy/tags.rego" (line 30, column 12):
  parse failed:

    30|   re_match(`^([A-Z][A-Z0-9\-]+)+`, a)
                   ^

  unexpected character '`'```

**Comment by jaspervdj-luminal** [comment]
Hi @sean-zou, thanks for the report!  This was indeed something we skipped for the initial implementation, but it should be very easy to add -- I've given it a stab in #140.

**Comment by jaspervdj-luminal** [comment]
This should be fixed in `v0.6.0`.  Let me know if it doesn't work!

**Comment by sean-zou** [comment]
It works now. Thanks for the quick fix!
